cmd/terramate/e2etests/run_test.go

```diff
@@ -385,6 +399,40 @@ stack-z
 				StderrRegex: string(dag.ErrCycleDetected),
 			},
 		},
+		{
+			name: "child stack can have explicit after clause to parent",
+			layout: []string{
+				`s:stacks`,
+				`s:stacks/child:after=["../"]`,
```

Comment:
```suggestion
				`s:stacks/child:after=["/stacks"]`,
```

to be consistent with the other tests that also use the abspath. Feel free to ignore.